doc/howto/storage_configure_pool.md

```diff
@@ -0,0 +1,21 @@
+# How to configure storage pool settings
+
+See the {ref}`storage-drivers` documentation for the available configuration options for each storage driver.
+
+General keys for a storage pool (like `source`) are top-level.
+Driver-specific keys are namespaced by the driver name.
+
+(storage-configure-pools)=
+## Configure storage pools
```

Comment:
No need for this subheading, since there is only one section and the title is the same as the page title.
The ID should go at the top of the file.